controllers/content_items_controller.go

```diff
@@ -0,0 +1,38 @@
+package controllers
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/alphagov/publishing-api/contentstore"
+	"github.com/alphagov/publishing-api/urlarbiter"
+)
+
+type ContentItemsController struct {
+	arbiter           *urlarbiter.URLArbiter
+	liveContentStore  *contentstore.ContentStoreClient
+	draftContentStore *contentstore.ContentStoreClient
+}
+
+func NewContentItemsController(arbiterURL, liveContentStoreURL, draftContentStoreURL string) *ContentItemsController {
+	return &ContentItemsController{
+		arbiter:           urlarbiter.NewURLArbiter(arbiterURL),
+		liveContentStore:  contentstore.NewClient(liveContentStoreURL),
+		draftContentStore: contentstore.NewClient(draftContentStoreURL),
+	}
+}
+
+func (c *ContentItemsController) PutDraftContentItem(w http.ResponseWriter, r *http.Request) {
+	registerWithURLArbiterAndForward(c.arbiter, w, r, func(basePath string, requestBody []byte) {
+		doContentStoreRequest(c.draftContentStore, "PUT", strings.Replace(basePath, "draft-", "", -1), requestBody, w)
```

Comment:
@alext: good eye! fixed in https://github.com/alphagov/publishing-api/commit/41768c653859a030be541cd77d76dbbeb3c080dd.